app/partials/navigation.jade

```diff
@@ -12,7 +12,7 @@
     .navbar-collapse.collapse(ng-class="{'in bg-blue' : !navCollapsed}", ng-click="navCollapsed=true")
       ul.nav.navbar-nav.navbar-right(ng-show="status.isLoggedIn")
         li.item(ng-show="settings.btcCurrency")
-          a.flex-important.flex-column(ng-click="openBCmarkets()")
+          a.flex-important.flex-column(ng-click="openBCmarkets()" href="")
```

Comment:
Ah I see, if it's in specific places then this makes sense. Though it might be more clear for other developers is you add a directive like `selectable`.